.ci.yaml

```diff
@@ -4506,25 +4506,6 @@ targets:
         ["devicelab", "hostonly", "mac", "arm64"]
       task_name: ios_app_with_extensions_test
 
-  - name: Mac_x64_ios ios_content_validation_test
```

Comment:
do we need to test this on both arches?

Reply:
Also, do tool integration tests run on an arbitrary arch?

Reply:
The tool integration test ios_content_validation_test was already running in  `host_cross_arch.shard` so it will run on both.

Reply:
oh yeah!